From e269eab2c8181161aa956675742859e204e351eb Mon Sep 17 00:00:00 2001
From: "Holesch, Simon (BSH)" <simon.holesch@bshg.com>
Date: Thu, 16 Jul 2020 19:07:42 +0200
Subject: [PATCH 2/2] Add startup delay

This is a workaround for synchronization problems between wifi driver
and connman.
---
 src/connman.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/connman.service.in b/src/connman.service.in
index 59c4d85..463773c 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -11,6 +11,7 @@ Wants=network.target
 Type=dbus
 BusName=net.connman
 Restart=on-failure
+ExecStartPre=/bin/sleep 3
 ExecStart=@sbindir@/connmand -n
 StandardOutput=null
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE CAP_NET_RAW CAP_SYS_TIME CAP_SYS_MODULE CAP_WAKE_ALARM
-- 
2.7.4

